2024-11-21 19:00:17,326 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Current SDK version is 0.18.0
2024-11-21 19:00:17,327 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Configure stats pid to 57560
2024-11-21 19:00:17,327 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Loading settings from /root/.config/wandb/settings
2024-11-21 19:00:17,327 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Loading settings from /mnt/srv.nas/Energy-Optimal-Inferencing/wandb/settings
2024-11-21 19:00:17,327 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-11-21 19:00:17,327 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-11-21 19:00:17,327 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-21 19:00:17,328 INFO    MainThread:57560 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-11-21 19:00:17,328 INFO    MainThread:57560 [wandb_init.py:_log_setup():525] Logging user logs to /mnt/srv.nas/Energy-Optimal-Inferencing/wandb/run-20241121_190017-dn3y309n/logs/debug.log
2024-11-21 19:00:17,328 INFO    MainThread:57560 [wandb_init.py:_log_setup():526] Logging internal logs to /mnt/srv.nas/Energy-Optimal-Inferencing/wandb/run-20241121_190017-dn3y309n/logs/debug-internal.log
2024-11-21 19:00:17,328 INFO    MainThread:57560 [wandb_init.py:_jupyter_setup():471] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f2d98c33a30>
2024-11-21 19:00:17,330 INFO    MainThread:57560 [wandb_init.py:init():609] calling init triggers
2024-11-21 19:00:17,330 INFO    MainThread:57560 [wandb_init.py:init():616] wandb.init called with sweep_config: {}
config: {}
2024-11-21 19:00:17,330 INFO    MainThread:57560 [wandb_init.py:init():659] starting backend
2024-11-21 19:00:17,330 INFO    MainThread:57560 [wandb_init.py:init():663] setting up manager
2024-11-21 19:00:17,333 INFO    MainThread:57560 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-21 19:00:17,335 INFO    MainThread:57560 [wandb_init.py:init():671] backend started and connected
2024-11-21 19:00:17,349 INFO    MainThread:57560 [wandb_run.py:_label_probe_notebook():1345] probe notebook
2024-11-21 19:00:17,350 INFO    MainThread:57560 [wandb_init.py:init():766] updated telemetry
2024-11-21 19:00:17,437 INFO    MainThread:57560 [wandb_init.py:init():799] communicating run to backend with 90.0 second timeout
2024-11-21 19:00:17,784 INFO    MainThread:57560 [wandb_init.py:init():850] starting run threads in backend
2024-11-21 19:00:18,332 INFO    MainThread:57560 [wandb_run.py:_console_start():2466] atexit reg
2024-11-21 19:00:18,332 INFO    MainThread:57560 [wandb_run.py:_redirect():2312] redirect: wrap_raw
2024-11-21 19:00:18,332 INFO    MainThread:57560 [wandb_run.py:_redirect():2377] Wrapping output streams.
2024-11-21 19:00:18,333 INFO    MainThread:57560 [wandb_run.py:_redirect():2402] Redirects installed.
2024-11-21 19:00:18,337 INFO    MainThread:57560 [wandb_init.py:init():893] run started, returning control to user process
2024-11-21 19:00:18,343 INFO    MainThread:57560 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-11-21 19:00:18,343 INFO    MainThread:57560 [wandb_init.py:_pause_backend():436] pausing backend
2024-11-21 19:10:21,706 INFO    MainThread:57560 [wandb_init.py:_resume_backend():441] resuming backend
2024-11-21 19:10:21,824 INFO    MainThread:57560 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-11-21 19:10:21,824 INFO    MainThread:57560 [wandb_init.py:_pause_backend():436] pausing backend
2024-11-21 19:10:23,748 INFO    MainThread:57560 [wandb_init.py:_resume_backend():441] resuming backend
2024-11-21 19:10:23,760 INFO    MainThread:57560 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-11-21 19:10:23,760 INFO    MainThread:57560 [wandb_init.py:_pause_backend():436] pausing backend
2024-11-21 19:10:39,966 WARNING MsgRouterThr:57560 [router.py:message_loop():77] message_loop has been closed
